internal/service/advancedcluster/model_advanced_cluster_test.go

```diff
@@ -150,6 +151,48 @@ func TestFlattenReplicationSpecs(t *testing.T) {
 	}
 }
 
+func TestGetDiskSizeGBFromReplicationSpec(t *testing.T) {
+	diskSizeGBValue := 40.0
+	hardwareSpec := admin.HardwareSpec20250101{
+		DiskSizeGB: admin.PtrFloat64(diskSizeGBValue),
+	}
+	regionConfig := []admin.CloudRegionConfig20250101{{
```

Comment:
knit: i wouldn't create vars for hardwareSpec, regionConfig and regionConfig, and would use inline

Reply:
adjusted, thanks